provisioning/miniprov/hare_mp/main.py

```diff
@@ -593,13 +785,14 @@ def main():
                        help_str='Initializes Hare',
                        handler_fn=init))
 
-    add_param_argument(
-        add_plan_argument(
-            add_file_argument(
-                add_subcommand(subparser,
-                               'test',
-                               help_str='Tests Hare component',
-                               handler_fn=test_IVT))))
+    add_dev_argument(
```

Comment:
Do you like this stairway of functions?
If not, we could make use of `functools` to improve readability. Example: https://mathieularose.com/function-composition-in-python

Reply:
I have not created it, I used it, to add my dev argument in IVT framework.

Reply:
But wouldn't you improve it given that you touched this piece of code anyway?

Reply:
Sure will take it and do this change